packages/server/src/server.ts

```diff
@@ -31,7 +31,11 @@ const buildApp = async (): Promise<Express> => {
     app.use(express.json({ limit: '50mb' }));
     app.use(compression());
 
-    const dotcomDevOrigins = ['http://localhost:3030', 'http://localhost:9000'];
+    const dotcomDevOrigins = [
+        'https://r.thegulocal.com',
```

Comment:
Is it worth linking through? And maybe explain that this is used for BrowserStack incase someone wants to remove it?

```suggestion
        // see: https://github.com/guardian/dotcom-rendering/blob/f05969110b0ab9af18041bbe537f93f98d28ad8f/dotcom-rendering/scripts/nginx/setup.sh#L11
        'https://r.thegulocal.com',
```